app/pages/docs/index.tsx

```diff
@@ -17,48 +17,55 @@ import {
   TableSpecimen,
   TypeSpecimen,
 } from "catalog";
-import { useEffect, useState } from "react";
-import { i18n, I18nProvider, parseLocaleString } from "../src";
+import { useEffect, useMemo, useState } from "react";
+import { i18n, I18nProvider, parseLocaleString } from "../../src";
+import Slugger from "github-slugger";
+
+const slugger = new Slugger();
 
 const pages: ConfigPageOrGroup[] = [
-  { path: "/", title: "Introduction", content: require("../docs/index.mdx") },
+  {
+    path: "/",
+    title: "Introduction",
+    content: require("../../docs/index.mdx"),
```

Comment:
Maybe you can set up paths in tsconfig and then use absolute paths in import declarations / require calls.

```suggestion
    content: require("@/docs/index.mdx"),
```

After changing in tsconfig:

```
    "paths": {
      "@/components/*": ["./components/*"],
      "@/docs/*": ["./docs/*"]
    }
```

Reply:
Yes, that was something that was bothering me for a while actually. Did it in [`e713737` (#476)](https://github.com/visualize-admin/visualization-tool/pull/476/commits/e713737036415d4eae50dd8c8b68299aefc3728d), thanks for the suggestion.